client/components/happychat/composer.jsx

```diff
@@ -11,7 +11,8 @@ import { isEmpty } from 'lodash';
 /**
  * Internal dependencies
  */
-import { sendChatMessage, setChatMessage } from 'state/happychat/actions';
+import { sendChatMessage } from 'state/happychat/actions';
+import { setChatMessage } from 'state/happychat/chat/actions';
```

Comment:
What is the rule on which we base this separation in `happychat/actions` and `happychat/chat/actions` ?